public_html/wp-content/mu-plugins/blocks/source/blocks/live-schedule/block.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable require-jsdoc */
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { getDataFromAPI } from './data';
+import LiveSchedule from './components/schedule';
+
+/**
+ * Display the live schedule.
+ *
+ * @returns {Component}
+ */
+class Block extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			sessions: [],
+			isFetching: true,
+		};
+		this.update = this.update.bind( this );
+		this.interval = setInterval( this.update, 60 * 1000 ); // 60 seconds in ms.
```

Comment:
We could tweak this a little, have it only fetch the data from the API every 10min or so but update the schedule display every minute. The schedule probably isn't changing much during the event, a 5-10min wait for fresh data would probably be fine - but we would still want the display to update quickly so when a session ends, the next is immediately visible. Thoughts?

Reply:
If the API endpoint is cached on at the server/network layer -- see https://github.com/WordPress/wordcamp.org/issues/204#issuecomment-535231721 -- then I think leaving it this way is fine, and simplest from an application perspective.

If not, though, then I agree that it'd be ideal to fetch new data every 5-10 min, but have the client update the interface every minute, regardless of whether new data has come in.

`MUST` - If the endpoint isn't cached at the network layer, then changing this to only fetch every 5+ min should probably be a blocker, to avoid a situation similar to the Tagregator bug at WCUS several years ago.

Reply:
> If the API endpoint is cached on at the server/network layer -- see #204 (comment) -- then I think leaving it this way is fine, and simplest from an application perspective.

I think it's a good idea to cache this at the server side, and eventually see if we can get the `If-Modified-Since` headers working (since this would still require a server hit, it won't protect against the tagregator issue alone). Let's move any more API cache discussion over to #207.

In any case, I've updated this PR to fetch the API every 5 minutes, while still re-rendering every minute.